CMSgov/design-system
# [WNMGDS-980] - Convert UsaBanner component to typescript

## Note merge request destination is a temporary `typescript-conversions` branch to be merged next sprint 

## Summary
Converted UsaBanner to a typescript FunctionComponent, updated test snapshots and docs site reference.

### Changed
Added json importing setting to global .tsconfig

### Removed
Removed jsx version of component

## How to test
Run `yarn build && yarn start` to confirm component compiles and works as before on docs site
Run `yarn storybook` to confirm component works as expected in storybook
Run `yarn test` to confirm no tests are broken
